src/main.nf

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env nextflow
+
+/*
+WGS Structural Variations Pipeline
+*/
+
+
+// 0. Preflight checks
```

Comment:
Hmm ... I mis-read "pre-flight as pref-light" first, and was confused ... its a nitpick, but maybe can clarify anyway? :)